src/Uno.UWP/Data/Pdf/PdfPage.Android.cs

```diff
@@ -0,0 +1,164 @@
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Android.Graphics;
+using Android.Graphics.Pdf;
+using Uno;
+using Windows.Foundation;
+using Windows.Storage.Streams;
+using Windows.UI;
+using Color = Windows.UI.Color;
+using Rect = Windows.Foundation.Rect;
+
+namespace Windows.Data.Pdf;
+
+public partial class PdfPage : IDisposable
+{
+	private readonly PdfRenderer.Page pdfPage;
+	private const float pointToPixelFactor = 96.00000000f / 72.0000000f;
+
+	internal PdfPage(PdfRenderer.Page pdfPage)
+	{
+		this.pdfPage = pdfPage ?? throw new ArgumentNullException(nameof(pdfPage));
+	}
+
+	[NotImplemented]
+	public PdfPageDimensions Dimensions { get; } = new PdfPageDimensions();
+
+	public uint Index => (uint)pdfPage.Index;
+
+	[NotImplemented]
+	public float PreferredZoom
+	{
+		get
+		{
+			throw new NotImplementedException("The member float PdfPage.PreferredZoom is not implemented in Uno.");
+		}
+	}
```

Comment:
```suggestion
	public float PreferredZoom 
	   => throw new NotImplementedException("The member float PdfPage.PreferredZoom is not implemented in Uno.");
```

Same for the similar others.